### PR TITLE
feat: added ip addresses to agent data source

### PR DESCRIPTION
**Pull request by joaomper-TE**
As requested within #128 by several users this adds ip addresses to the Agent DataSource:

```
Changes to Outputs:
  + agent_id     = 123213
  + agent_name   = "Vancouver, Canada"
  + ip_addresses = [
      + "155.313.108.19",
      + "155.313.108.23",
      + "155.313.108.15",
      + "2607:f6f3:3400:96::2:7",
      + "2607:f6f3:3400:96::2:4",
    ]
╷
```

### CODE REVIEW

File: docs/data-sources/account_group.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ resource "thousandeyes_http_server" "thousandeyes_http_test" {
   }
 
   agents {
-    agent_id = 3 # Singapur
+    agent_id = 3 # Singapure
```
Comment (pedro-te) [inline docs/data-sources/account_group.md:30]: It's actually `Singapore` 😄

Comment (joaomper-TE) [reply docs/data-sources/account_group.md:30]: 😄 will correct!

**Comment by joaomper-TE** [comment]
/review @phpinhei-te @brumarqu-te @sfreitas-te @shahid-te @vascmart-te @jabrowne-te

**Review by vascmart-te — APPROVED** [review]

**Review by phpinhei-te — APPROVED** [review]

**Review by pedro-te — APPROVED** [review]

**Comment by joaomper-TE** [comment]
> i'm not sure but from the referenced issue seems that customer needs the publicIpAddresses.
We need to pull in the IP addresses of agent in Sao-paulo Brazil for example and then allow those IP's on our network firewall

Yeah, that's a fair question. I asked yesterday in the GH issue to make sure which IP Addresses the user is referring to (will only merge this afterwards)

**Review by sfreitas-te — CHANGES_REQUESTED** [review]
Please add the issue number to the log messages

**Pull request merged**